service/mapper/transitions_test.go

```diff
@@ -0,0 +1,1078 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package mapper
+
+import (
+	"io"
+	"sync"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/ledger/complete/mtrie/trie"
+	"github.com/onflow/flow-go/model/flow"
```

Comment:
Those three imports are from `onflow/flow-go` though? 🤔 Do we usually split `ledger` and `model`?

Reply:
Oh OK I looked at the file to see the context around it and you meant the two imports below. It's not easy to get it in GitHub because it shows only the highlighted lines 😄

Reply:
Yeah, sorry, thought about `flow-go` vs `flow-dps`. Could have placed the pin on the more appropriate line 😃